communication/utils.py

```diff
@@ -28,19 +28,17 @@ def get_autologin_link(unique_token):
 class SmsSender:
 
     def send(self, send_to, send_from, message):
-        url = "https://junebug.qa.dig-it.me/jb/channels/ab79e9c5-7d15-440f-8bd0-6a0e2d2445b1/messages/"
+        url = settings.JUNEBUG_BASE_URL + settings.JUNEBUG_CHANNEL_ID
```

Comment:
This is not equivalent to the above, it would result in `https://junebug.qa.dig-it.me/jb/channels/ab79e9c5-7d15-440f-8bd0-6a0e2d2445b1` instead of `https://junebug.qa.dig-it.me/jb/channels/ab79e9c5-7d15-440f-8bd0-6a0e2d2445b1/messages/`.

We probably should also use `urljoin` here instead of string concatenation, to handle the cases where there are missing or excess `/`s at the beginning/end of strings, eg. if the base URL is `/jb/channels` and the channel id is `abc-456`, then we'll end up with a URL of `/jb/channelsabc-456` instead of `jb/channels/abc-456`.